docs/reference/api/usb/USBKeyboard.md

```diff
@@ -0,0 +1,31 @@
+## USBKeyboard
+
+<span class="images">![](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_keyboard.png)<span>USBKeyboard class hierarchy</span></span>
+
+The USBKeyboard class provides the functionality of a keyboard. You can send key presses, check the status of control keys and send a key press sequences though a stream interface. If you need mouse or keyboard and mouse functionality, please see the [USBMouse](USBMouse.html) and [USBMouseKeyboard](USBMouseKeyboard.html) classes.
```

Comment:
What are some best practices for using USBKeyboard? What are some concepts you need to understand to use it?